### PR TITLE
Add bond config to virtualmedia workflow

### PR DESCRIPTION
**Pull request by creydr**
In #33050 we added a job for e2e tests on baremetalds for a bonded primary interface. This PR is the follow up on https://github.com/openshift/release/pull/33050#issuecomment-1280522278 to have this tested in a more regular basis. 

**Hints for reviewers:**
The adding of the bonds to the `baremetalds-e2e-sdn-serial-virtualmedia` workflow was done in ab3f35c417c0ff007ec6a9a18cfc54ce00b342db. The other commit (300783ea9e171e8f978916e3929c1d8dd6a07e19) was only to rename the workflow to have the `-bond` suffix to make it clear that this workflow is based on a bonded primary interface.

/cc @derekhiggins @cybertron

### CODE REVIEW

**Comment by creydr** [comment]
Hello @derekhiggins ,
could you PTAL at this PR? It is the followup to your comment in https://github.com/openshift/release/pull/33050#issuecomment-1280522278
Thanks a lot in advance

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.12.yaml [hunk 1]
```diff
@@ -196,7 +196,7 @@ tests:
   interval: 48h
```
Comment (derekhiggins) [inline ci-operator/config/openshift/release/openshift-release-master__nightly-4.12.yaml:196]: > - as: e2e-metal-ipi-sdn-serial-virtualmedia
I wonder should we also rename this to "e2e-metal-ipi-sdn-serial-virtualmedia-bond"
so its more obvious to the casual observer ?

Comment (creydr) [reply ci-operator/config/openshift/release/openshift-release-master__nightly-4.12.yaml:196]: Yeah. Probably makes sense too. Let me update it...

Comment (creydr) [reply ci-operator/config/openshift/release/openshift-release-master__nightly-4.12.yaml:196]: Updated it to include the suffix

**Comment by creydr** [comment]
@wking and @jupierce could you PTAL at this PR. I guess we need both of your approvals, as we rename one of the jobs.
Thanks a lot in advance

**Pull request merged**